studentvue/studentvue.py

```diff
@@ -237,6 +237,33 @@ def get_class_info(self, class_):
 
         return self._parse_grade_book_class_page(grade_book_class_page, class_.name)
 
+    def get_course_history(self):
```

Comment:
I can look into it more later, but for me, it's only getting the first semester.
```
{
   '9':[
      [
         CCSS Geometry B (GEOC150B) with grade of A,
         Chinese 3B H (CHIN253B) with grade of A,
         College/Career (COCA150) with grade of A,
         English 1B (ENGL151B) with grade of A,
         NGSS Biology B (BION150B) with grade of A,
         PE 1B (PEPE151B) with grade of A,
         World History B (WHST150B) with grade of A
      ]
   ],
   '10':[
      [
         AP Chinese B (CHIN350B) with grade of A,
         AP Euro Hist B (EUHI350B) with grade of A,
         CCSS Algebra 2B (ALGC152B) with grade of A,
         Chemistry B H (CHEM250B) with grade of A,
         Comp Prg B (PROG150B) with grade of A,
         English 2B (ENGL152B) with grade of A,
         Fit for Life 2B (FITL152B) with grade of A
      ]
   ]
}
```

Reply:
Silly me, it was an indentation error I didn't catch. It works fine now (at least for me)
```python
{
  '6': [
    [Homeroom(HMRM050) with grade of N, Japanese 1(JAPN051) with grade of A,
      Language Arts 6 Honors(LANG256) with grade of A,
      Math 6 Honors(MATH256) with grade of A,
      Physical Education 6(PEPE056) with grade of B,
      Science 6 Honors(SCIE256) with grade of A,
      Social Studies 6 Honors(SOCS256) with grade of A],
    [Homeroom(HMRM050) with grade of N, Japanese 1(JAPN051) with grade of A,
      Language Arts 6 Honors(LANG256) with grade of B,
      Math 6 Honors(MATH256) with grade of A,
      Physical Education 6(PEPE056) with grade of A,
      Science 6 Honors(SCIE256) with grade of A,
      Social Studies 6 Honors(SOCS256) with grade of A]
  ],
  '7': [
    [CCSS Math 7(CCMA057) with grade of A,
      Homeroom(HMRM050) with grade of N,
      Japanese 2(JAPN052) with grade of A,
      Language Arts 7 Honors(LANG257) with grade of A,
      Physical Education 7(PEPE057) with grade of A,
      Science 7 Honors(SCIE257) with grade of A,
      Social Studies 7 Honors(SOCS257) with grade of B],
    [CCSS Math 7(CCMA057) with grade of A,
      Homeroom(HMRM050) with grade of N,
      Japanese 2(JAPN052) with grade of A,
      Language Arts 7 Honors(LANG257) with grade of A,
      Physical Education 7(PEPE057) with grade of B,
      Science 7 Honors(SCIE257) with grade of A,
      Social Studies 7 Honors(SOCS257) with grade of A]
  ],
...
]
```